app/views/privacy_policies/show.html.erb

```diff
@@ -1,23 +1,22 @@
 <% content_for :title, "Privacy policy" %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-three-quarters">
-    <h1 class="govuk-heading-xl">Privacy policy</h1>
+    <div class="govuk-grid-column-two-thirds">
 
-    <% if !@policy.acceptance_required?(current_user) %>
-      <% content_for :before_content, govuk_back_link(text: "Back", href: :back) %>
-    <% else %>
-      <p>Before you continue, please check our privacy policy</p>
-      <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
```

Comment:
Are we ok removing this condition?

Reply:
Well the condition is still there other than the line ( Before you continue and the <hr> ) the rest should still be there. 

Not sure what the line of content saying that does for a user to be fair. Condition for acceptance is still in place isnt it @jamesrose